x/concentrated-liquidity/math/math_test.go

```diff
@@ -70,6 +70,13 @@ func (suite *ConcentratedMathTestSuite) TestLiquidity0() {
 			expectedLiquidity: "1519437308.014768571720923239",
 			// https://www.wolframalpha.com/input?i=1000000+*+%2870.710678118654752440*+74.161984870956629487%29+%2F+%2874.161984870956629487+-+70.710678118654752440%29
 		},
+		"sqrtPriceA greater than sqrtPriceB": {
```

Comment:
If we had more test cases, we could always test that permuting the args gives the same result.

Reply:
Note to self: come back and check for rounding concerns with this fn